appveyor.yml

```diff
@@ -1,47 +1,52 @@
 # Set up the build environment.
 environment:
-    # WORKAROUND START #
-    RUSTUP_USE_HYPER: 1
-    CARGO_HTTP_CHECK_REVOKE: false
-    # WORKAROUND END #
-
-    matrix:
-        - channel: stable
-          target: x86_64-pc-windows-msvc
-        - channel: beta
-          target: x86_64-pc-windows-msvc
-        - channel: nightly
-          target: x86_64-pc-windows-msvc
-
+  matrix:
+    - channel: stable
+      target: x86_64-pc-windows-msvc
+    - channel: stable
+      target: x86_64-pc-windows-gnu
+    - channel: beta
+      target: x86_64-pc-windows-msvc
+    - channel: beta
+      target: x86_64-pc-windows-gnu
+    - channel: nightly
+      target: x86_64-pc-windows-msvc
+    - channel: nightly 
+      target: x86_64-pc-windows-gnu
+
+# Allow for occasional compiler breakage on nightly Rust.
 matrix:
-    allow_failures:
-        - channel: nightly
+  fast_finish: true
+  allow_failures:
+    - channel: nightly
 
 # Set up the Rust toolchain.
 install:
-    - set PATH=C:\Program Files\Git\mingw64\bin;%PATH%
-    - curl -fsS "https://win.rustup.rs/" -o rustup-init.exe
-    - rustup-init.exe -y --default-toolchain %channel%-%target%
-    - set PATH=%PATH%;C:\Users\appveyor\.cargo\bin
-    - rustc -V
-    - cargo -V
+  - set PATH=C:\Program Files\Git\mingw64\bin;%PATH%
+  - curl -fsS "https://win.rustup.rs/" -o rustup-init.exe
+  - rustup-init.exe -y --default-toolchain %channel%-%target%
+  - set PATH=%PATH%;C:\Users\appveyor\.cargo\bin
+  - rustc -V
+  - cargo -V
+
+# Build the Bors staging and trying branches, plus PRs to develop and master.
+branches:
+  only:
+    - staging
+    - trying
+    - develop
+    - master
```

Comment:
Should also build the latest release branch.

Reply:
Ah, good catch. Will add this now.